from the configure script:

> We need 'closefrom' to avoid a race in reentrant multi-threaded writing of
> compressed files.  This is due to copying filedescriptors during fork,
> which is needed to support safe writing of compressed files through a pipe
> and the help of an external compression utility (such as 'gzip').

and indeed this one test is incredibly racy

also see https://github.com/arminbiere/cadical/issues/117

diff -ruN a/test/api/run.sh b/test/api/run.sh
--- a/test/api/run.sh	2025-02-10 16:43:14.000000000 +0100
+++ b/test/api/run.sh	2025-06-01 13:25:05.809200875 +0200
@@ -101,7 +101,6 @@
 
 #--------------------------------------------------------------------------#
 
-run parcompwrite
 
 run newdelete
 run unit
